docs/api/identity-service.md

```diff
@@ -162,10 +162,6 @@ The `type` query parameter specifies the identity type to return. Accepted value
         "deviceId": "device01",
         "moduleId": "module01",
         "genId": "12345",
-        "auth": {
-            "type": "sas",
-            "keyHandle": "string"
```

Comment:
So list does not include auth info, but get does? Why is that?

Reply:
Yeah, that was at the root of my question when I opened #56.  @arsing?